src/lib.rs

```diff
@@ -266,8 +274,14 @@ impl CreateOptions {
 	///
 	/// Support for this feature and specific sizes depend on the CPU and kernel configuration.
 	/// See also: <https://www.kernel.org/doc/html/latest/admin-guide/mm/hugetlbpage.html>
-	pub fn huge_tlb(&mut self, value: Option<HugeTlb>) -> &mut Self {
-		self.huge_table = value;
+	pub fn huge_tlb(mut self, value: HugeTlb) -> Self {
+		self.huge_table = Some(value);
+		self
+	}
+
+	/// Disable the huge TLB option set by [`Self::huge_tlb`].
+	pub fn no_huge_tlb(mut self) -> Self {
+		self.huge_table = None;
```

Comment:
This could be write:

```rrust
pub fn huge_tlb<T: Into<Option<HugeTlb>>>(mut self, value: T) -> Self {
    self.huge_table = value.into();
    self
}
```

This has big advantage to be totally invisible to the user while allowing thing like `CreateOptions::new().huge_tlb(take_the_decision())`

Reply:
Yeah, already updated it to `impl Into<Option<HugeTlb>>` :)